starter-code/index.html

```diff
@@ -2,30 +2,63 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+
+  <link rel="stylesheet" href="./css/main.css">
 </head>
+
 <body>
-  Instagram
 
-  Capture and Share the World's Moments
+  <div class="flex flex-horizontal margin-general">
+    <img src="./images/phones.png" alt="">
+
+    <div class="content" >
+      <header class="flex flex-header"> 
+        <img src="./images/brand.png" alt="brand">
+
+        <button>
+          <img src="./images/home.png" alt="house">Log in
+        </button>
+      </header>
+
+      <section class="background-white">
+        <h1>
+          <strong>Capture and Share <br>the World's Moments </strong>
+        </h1>
+
+        <p>
+          Instagram is a <strong>fast, beautiful</strong> and <strong>fun</strong> way to share your life with friends and family.
+        </p>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+        <p>
+            Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+        </p>
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+        <p>
+            Oh yeah, did we mention it's free?
+        </p>
 
-  Oh yeah, did we mention it's free?
+        <div>
+          <img src="./images/badge-iphone.png" alt="Download on the App Store">
+          <img class = "sp-b-img" src="./images/badge-android.png" alt="Get it on Google Play">
+        </div>
 
-  Download on the App Store
-  Get it on Google Play
+      </section>
+    </div>
+  </div>
 
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
+  <footer>
+    <ul class="flex flex-header">
+      <li><a href="#">About Us</a></li>
+      <li><a href="#">Support</a></li>
+      <li><a href="#">Blog</a></li>
+      <li><a href="#">Press</a></li>
+      <li><a href="#">API</a></li>
+      <li><a href="#">Jobs</a></li>
+      <li><a href="#">Privacy</a></li>
+      <li><a href="#">Terms</a></li>
+      <li>&copy; 2014 Instagram</li>
```

Comment:
Apart from the little detail in the footer, everything looks perfectly align.

Well done!